content/docs/command/command_line_part2.md

```diff
@@ -13,3 +13,181 @@ description = "Goals"
 - Watch this video until 8:30 on [file paths](https://www.youtube.com/watch?v=BMT3JUWmqYY)
 - (Optional): watch this video on the [history of keyboards and the command line interface](https://www.youtube.com/watch?v=4RPtJ9UyHS0)
 - (Optional - for Windows users only): [install the Cmder console emulator](https://cmder.net/) (easier) or [set up the windows subsystem for linux](https://www.youtube.com/watch?v=LLlfLpvQg04)
+
+## Class Curriculum
+
+| Section content                                              | Expected time (mins) | Pre - Requirements                |
+| ------------------------------------------------------------ | -------------------- | --------------------------------- |
+| Kahoot                                                       | 15 minutes           | videos                            |
+| Questions from Kahoot + [Session 1](../command_line_part1)   | 15 minutes           | ❌                                |
+| Useful Terms/Concepts                                        | 5-10 minutes         | videos                            |
+| Navigate file system and working with files & folders (demo) | 10-15 minutes        | ❌                                |
+| Quick overview of package managers                           | 5-10 minutes         | ❌                                |
+| Class break                                                  | 10 minutes           | ❌                                |
+| Command line practice in groups                              | 40-50 minutes        | local terminal OR repl.it account |
+| Summary & Next class overview                                | 5 minutes            | ❌                                |
+
+## Useful Terms/Concepts
+
+- **File**: data, usually organized according to some specific _file format_ (e.g. `txt`, `mp3`, `docx`)
+- **Directory**: a file that stores information about the length and location of other files, similar to a table of contents
```

Comment:
Would it be more

> a folder that stores files or other directories

🤔

Reply:
will add that as well, but technically in linux-systems directories are just special files -- one of the videos they watched talks about this a bit so hopefully it'll tie to that a bit